src/content/platform-integration/web/wasm.md

```diff
@@ -29,19 +24,10 @@ Revisit this page for the latest updates.
   section to learn more. 
 :::
 
-[`beta`]: {{site.github}}/flutter/flutter/wiki/flutter-build-release-channels#beta
-[`main`]: {{site.github}}/flutter/flutter/wiki/flutter-build-release-channels#master-aka-main
+[`stable`]: {{site.github}}/flutter/flutter/wiki/flutter-build-release-channels#stable
 [`package:web`]: {{site.pub-pkg}}/web
 [`dart:js_interop`]: {{site.dart.api}}/{{site.dart.sdk.channel}}/dart-js_interop 
 
-:::note
```

Comment:
This shouldn't be linked from the Wasm page. It's kinda weird. We should put this maybe in the FAQ?

Reply:
Feel free to delete it. File a bug if you feel it should be put in the FAQ, cause I guarantee I will forget.

Reply:
https://github.com/flutter/website/issues/10554